pkg/gcv/configs/config.go

```diff
@@ -464,21 +538,71 @@ type Configuration struct {
 
 func loadRegoFiles(dir string) ([]string, error) {
 	var libs []string
+	var content []byte
 	files, err := ListRegoFiles(dir)
+
 	if err != nil {
 		return nil, errors.Wrapf(err, "failed to list rego files from %s", dir)
 	}
+
 	for _, filePath := range files {
 		glog.V(2).Infof("Loading rego file: %s", filePath)
-		fileBytes, err := ioutil.ReadFile(filePath)
+
+		if strings.HasPrefix(filePath, "gs://") {
```

Comment:
I'm seeing this if gs / else a lot.  I think it would be better factored to have an interface with a read method to wrap the file, then have a struct for GCS and a struct for local files that implement the interface.